src/Faker/ORM/Spot/ColumnTypeGuesser.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Faker\ORM\Spot;
+
+use Faker\Generator;
+
+class ColumnTypeGuesser
+{
+    protected $generator;
+
+
+    /**
+     * ColumnTypeGuesser constructor.
+     * @param Generator $generator
+     */
+    public function __construct(Generator $generator)
+    {
+        $this->generator = $generator;
+    }
+
+    /**
+     * @param array $field
+     * @return \Closure|null
+     */
+    public function guessFormat(array $field)
+    {
+        $generator = $this->generator;
+        $type = $field['type'];
+        switch ($type) {
+            case 'boolean':
+                return function () use ($generator) {
+                    return $generator->boolean;
+                };
+            case 'decimal':
+                $size = isset($field['precision']) ? $field['precision'] : 2;
+
+                return function () use ($generator, $size) {
+                    return $generator->randomNumber($size + 2) / 100;
+                };
+            case 'smallint':
+                return function () {
+                    return mt_rand(0, 65535);
+                };
+            case 'integer':
+                return function () {
+                    return mt_rand(0, intval('2147483647'));
```

Comment:
Copied it from Doctrine column type guesser. Will fix both today. 

I think ``$generator->randomNumber(10);`` is even more correct solution here...